aws-distributed-configuration/src/main/java/io/micronaut/aws/distributedconfiguration/AwsDistributedConfigurationClient.java

```diff
@@ -119,6 +121,14 @@ public Publisher<PropertySource> getPropertySources(Environment environment) {
         return Publishers.just(new MapPropertySource(propertySourceName, result));
     }
 
+    /**
+     *
+     * @param originalKey an original property key
+     * @param groupName a property group name
+     * @return An adapted property key (e.g. key that has been appended to a prefix)
+     */
+    protected abstract String adaptPropertyKey(String originalKey, String groupName);
```

Comment:
add `@since` and nullability annotations

Reply:
Done